.circleci/regenerate.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python3
+
+# regenrates config.yml based on config.yml.in
+
+from copy import deepcopy
+import os.path
+
+import jinja2
+import yaml
+from jinja2 import select_autoescape
+
+WORKFLOWS_JOBS_PR = {"filters": {"branches": {"ignore": ["master"]}}}
+
+WORKFLOWS_JOBS_MASTER = {
+    "context": "org-member",
+    "filters": {"branches": {"only": ["master"]}},
+}
+
+
+def indent(indentation, data_list):
+    return ("\n" + " " * indentation).join(
+        yaml.dump(data_list, default_flow_style=False).splitlines()
+    )
+
+
+def jobs(pr_or_master, num_workers=20, indentation=2):
+    w = {}
```

Comment:
What do w and d stand for?

Reply:
i stole it from vision, where i think it stood for workflow? and then i just kept using it for all of them
d is dict
i will get slightly better names